system/forms.py

```diff
@@ -0,0 +1,32 @@
+from django import forms
+from system.models import Patient, Location
+
+
+class PatientForm(forms.Form):
+    caseID = forms.IntegerField()
+    name = forms.CharField()
+    IDnum = forms.CharField()
+    DoB = forms.DateField()
+    DCC = forms.DateField()
+
+
+
+class LocationForm(forms.Form):
+
+    name = forms.CharField()
+    address = forms.CharField()
+    district = forms.CharField()
+    x = forms.IntegerField()
+    y = forms.IntegerField()
+
+
+
+class Visit(forms.Form):
```

Comment:
Class name is inconsistent with other classes in this module
```suggestion
class VisitForm(forms.Form):
```

Reply:
Solved in next pull